src/lib/analytics/GoodIdAnalytics.js

```diff
@@ -0,0 +1,14 @@
+import { osName, osVersion } from 'mobile-device-detect'
+import { version } from '../../../package.json'
+
+export const analyticsConfig = {
+  google: { enabled: true },
```

Comment:
we dont need google analytics id per app?

Reply:
No, it uses on web window.dataLayer and the idea is managed by the script in index.html.
for android it uses `"@react-native-firebase/analytics";` which does not seem to take a app-id.